stock_picking_show_linked/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
```

Comment:
Please improve licensing header by adding the AGPL good string (in other files too):

`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).`

Reply:
Hi @rousseldenis Thank you for your input I updated as per your suggestion in the following commit and squashed that commit.